scripts/travis/kafka-integration-test.sh

```diff
@@ -2,16 +2,19 @@
 
 set -e
 
-docker pull spotify/kafka
-CID=$(docker run -d -p 2181:2181 -p 9092:9092 --env ADVERTISED_HOST=localhost --env ADVERTISED_PORT=9092 --rm spotify/kafka)
+docker pull wurstmeister/kafka
```

Comment:
note that this is using kafka 2.x before it was 1.x.

`wurstmeister/kafka` seems to be an active project. The `spotify/kafka` seems to be unmaintained.

Reply:
Shouldn't we use the one from confluence? That one is definitely supported.

Reply:
There is also http://strimzi.io which is maintained by Red Hat :)

Reply:
@yurishkuro which one do you mean?  This docker pull confluentinc/cp-kafka? it seems it's packaged with more confluent specific stuff.

One thing I don't like about these other images that they follow their versioning which is different to kafka versions.